app/javascript/controllers/application.js

```diff
@@ -0,0 +1,9 @@
+import { Application } from "@hotwired/stimulus"
+
+const application = Application.start()
+
+// Configure Stimulus development experience
+application.debug = false
```

Comment:
true? not even sure what this does.

Reply:
It configures the Stimulus development experience, duh

(I have no idea either. I'll try to figure it out quickly!)

Reply:
Looks like this prints some console logs:

<img width="618" alt="Screen Shot 2022-03-28 at 8 24 18 PM" src="https://user-images.githubusercontent.com/115918/160508689-db96b6d0-5232-4007-8cd5-995d68968788.png">

Probably a little crufty if most people aren't using Stimulus, so maybe just something to know as an interviewer if someone does go the Stimulus route.